plugins/SLURM-Integration/ufm_slurm_epilog.py

```diff
@@ -40,9 +40,14 @@ def epilog_init(self):
                         Constants.TOKEN_AUTH, Constants.CONF_TOKEN))
                     sys.exit(self.should_fail)
                 self.session = self.ufm.getServerSession(auth_type=self.auth_type, token=token)
+            elif self.auth_type == Constants.KERBEROS_AUTH:
+                principal_name = self.general_utils.get_conf_parameter_value(Constants.CONF_PRINCIPAL_NAME)
+                principal_pass = self.general_utils.get_conf_parameter_value(Constants.CONF_PRINCIPAL_PASSWORD)
```

Comment:
password will be stored as text in config file?
I don't think so ... should be encrypted somehow

Reply:
one of the optionsto to obtain the TGT  is to use the keytab method, by this way the user will not need to provide the password.

Reply:
I have removed the usage of principal_pass from the conf file because it's not needed.